### DISCUSSION TITLE
Unable to verify school ID

### DISCUSSION THREAD

**Discussion by westhecool** [community/community#69553] [GitHub Education]
### Hi everyone,

I am homeschooled and don't have a "standard" school ID. I do have one that my mom made but the system is refusing to accept it. It is dated, has my name, date of birth, grade, and my mom's name.

**Comment by santoshvandari** [comment]


**Reply by westhecool** [reply]
![opera_M8J6CXwtQZ](https://github.com/community/community/assets/52859033/bdafee4a-742e-4eaa-9ad3-11112c9fd757)
I would try contacting the support but as you can see here it seems that they don't offer any support.
